### PR TITLE
Merge master-2.x commits from 2023/01/01-2023/01/15

### PR DESCRIPTION
**Pull request by jiacheliu3**
What changes are proposed in this pull request?
This PR merges all missing commits in 2023/01/01 ~ 2023/01/15 from `master-2.x` into `main`.

Except distributed load v2
https://github.com/Alluxio/alluxio/commit/db9f07a50e1f7f6c45d511d591c8775ada6b71bb

### CODE REVIEW

**Review by apc999 — APPROVED** [review]

File: .github/workflows/stale.yaml
Comment (Xenorith) [inline .github/workflows/stale.yaml:5]: @LuQQiu i'm guessing we should disable this cron from the master-2.x branch?

File: docs/en/operation/User-CLI.md
Comment (Xenorith) [inline docs/en/operation/User-CLI.md:1104]: @jja725 i'm assuming it is appropriate to keep the docs here because the load feature was added in `dora` and so it's already in `main`

**Review by Xenorith — APPROVED** [review]
non-java changes look good to me

**Review by elega — APPROVED** [review]

File: dora/shaded/client/pom.xml
Comment (dbw9580) [inline dora/shaded/client/pom.xml:349]: any reason we don't use `${project.parent.parent.parent.basedir}` syntax here?

File: dora/shaded/client/pom.xml
Comment (dbw9580) [inline dora/shaded/client/pom.xml:370]: cross reference with the comment above

**Review by dbw9580 — APPROVED** [review]
LGTM with a minor issue

File: dora/shaded/client/pom.xml
Comment (jiacheliu3) [reply dora/shaded/client/pom.xml:349]: i copied this somewhere from existing dora code, all i know is this works

File: dora/shaded/client/pom.xml
Comment (jiacheliu3) [reply dora/shaded/client/pom.xml:370]: yea this is just to change the minimal. I don't know maven well enough to refactor this. I guess we can change these two to the same format after my PR is merged? I'd like this change to be in a separate PR instead of a shady commit hidden in this one.

Comment (dbw9580) [reply dora/shaded/client/pom.xml:370]: sound good to me. not really a blocker for this PR.

**Comment by jiacheliu3** [comment]
@ssz1997 @jja725 I will merge this one first to unblock a few other changes. However, I believe your review will still be required and highly valued on this PR. I will resolve your comments in later PRs. Such MergeXXX PRs are easy to find and I can simply revisit your open comments later. Thanks!

**Comment by jiacheliu3** [comment]
alluxio-bot, sync-merge this please

**Pull request merged**
